source/03_development/10_plugin_system/index.rst

```diff
@@ -171,23 +171,42 @@ The manifest entry ``Plugin-Class`` as the entrypoint of the plugin has two vari
 * If the full WebApplicationContext is not required - the simplified default ``org.ehrbase.plugin.NonWebMvcEhrBasePlugin`` must be implemented
   as in the `simple example plugin <https://github.com/ehrbase/ExamplePlugin/blob/master/simple-plugin/src/main/java/org/ehrbase/example_plugin/ExamplePlugin.java>`_.
 
-For both versions, it is recommended to create a spring configuration class triggering a ComponentScan on the plugin package. 
+For both versions, it is recommended to create a spring configuration class triggering a ComponentScan on the plugin package.
 With this setup, the boilerplate of the plugin implementation is done and the specific plugin logic may be implemented.
 
 Extension points
 ----------------
 
-While it might be enough to have additional code running in parallel and based on EHRbase, but the plugin use-case may also 
+While it might be enough to have additional code running in parallel and based on EHRbase, but the plugin use-case may also
 require hooking into processes and services, and react on or modify data.
 
 To handle this requirements additional extension point interfaces are provided by the plugin SPI, which provide aspect-like hooks on specific functionalities.
-
-Available extension points are in the package ``org.ehrbase.plugin.extensionpoints`` or in 
+Available extension points are in the package ``org.ehrbase.plugin.extensionpoints`` or in
 `GitHub <https://github.com/ehrbase/ehrbase/tree/develop/plugin/src/main/java/org/ehrbase/plugin/extensionpoints>`_,
 and the respective JavaDoc should be consulted on the specific extension point hook.
 
-To implement an extension point, create an implementation of the specific extension point interface as a component of the plugin, 
-and annotate it with ``@Extension`` and a specific ``@Order(#)`` annotation to define the execution order of the implementation.
+To implement an extension point, create an implementation of the specific extension point interface as a component of the plugin,
+and annotate it with ``@Extension`` and a specific ``@Order(#)`` annotation or implement the Order or PriorityOrder interface to define the execution order of the implementation.
+Please be aware that the lowest order value has the highest precedence, meaning it can process the input value first, but will process the return value last (same as with around pointcuts in aspects).
```

Comment:
Generally: "Around advice" with upper case instead of "around pointcuts"?

Reply:
yeah. i think that would be better, since advice is also the term used in the spring AOP docs afaik